cafebabel/static/js/script.js

```diff
@@ -113,3 +113,38 @@ Array.from(document.querySelectorAll('h1.edit input')).forEach(inputh1 => {
     () => inputh1.value || h1.classList.remove('active')
   )
 })
+
+/* Auto-expand textarea when typing http://jsfiddle.net/hmelenok/WM6Gq/ */
+var textAreaAutoResize
+if (window.attachEvent) {
+  textAreaAutoResize = function(element, event, handler) {
+    element.attachEvent('on' + event, handler)
+  }
+} else {
+  textAreaAutoResize = function(element, event, handler) {
+    element.addEventListener(event, handler, false)
+  }
+}
+function textAreaResizing() {
+  Array.from(document.querySelectorAll('textarea')).forEach(textArea => {
+    function resize() {
```

Comment:
Not certain why we need to declare this function inside of the forEach 🤔

Reply:
I worked only on the first `<textarea>` otherwise

Reply:
yes, but if you declare the function outside and you pass the `textArea` as an argument.
I suppose there was a reason but it looks very unsusual. Maybe it has to do with the setTimeout. (?)

Reply:
I can't get it work else way :-(